model-server-test/src/test/kotlin/IgnitePostgresTest.kt

```diff
@@ -35,6 +36,18 @@ class IgnitePostgresTest {
         store.dispose()
     }
 
+    @Test
+    fun `load multiple keys into cache`() {
```

Comment:
Can this be a bit more explicit on which condition should be fulfilled? Currently, I find it hard to understand what is actually tested here.

Reply:
Done.
Is it now better to understand?